lib/connections/limiter.go

```diff
@@ -73,22 +146,55 @@ func (lim *limiter) CommitConfiguration(from, to config.Configuration) bool {
 
 	lim.limitsLAN.set(to.Options.LimitBandwidthInLan)
 
+	// Set limits for devices
+	for _, v := range to.Devices {
+		if v.DeviceID == lim.myID {
+			// This limiter was created for local device. Should skip this device
+			continue
+		}
+
+		readLimiter, _ := lim.deviceReadLimiters.Load(v.DeviceID)
+		if v.MaxRecvKbps <= 0 {
+			readLimiter.(*rate.Limiter).SetLimit(rate.Inf)
+		} else {
+			readLimiter.(*rate.Limiter).SetLimit(1024 * rate.Limit(v.MaxRecvKbps))
+		}
+
+		writeLimiter, _ := lim.deviceWriteLimiters.Load(v.DeviceID)
+		if v.MaxSendKbps <= 0 {
+			writeLimiter.(*rate.Limiter).SetLimit(rate.Inf)
+		} else {
+			writeLimiter.(*rate.Limiter).SetLimit(1024 * rate.Limit(v.MaxSendKbps))
+		}
+
+		sendLimitStr := "is unlimited"
+		recvLimitStr := "is unlimited"
+		if v.MaxSendKbps > 0 {
+			sendLimitStr = fmt.Sprintf("limit is %d KiB/s", v.MaxSendKbps)
+		}
+
+		if v.MaxRecvKbps > 0 {
+			recvLimitStr = fmt.Sprintf("limit is %d KiB/s", v.MaxRecvKbps)
+		}
+		l.Infof("Device %s: send rate %s, receive rate %s", v.DeviceID, sendLimitStr, recvLimitStr)
```

Comment:
This will always print stuff, even if someones rate hasn't changed since before.
It will further always mess around with the rate limiters, regardless if the value changed or not.